src/GoTrueClient.ts

```diff
@@ -286,6 +286,15 @@ export default class GoTrueClient {
         if (error) {
           this._debug('#_initialize()', 'error detecting session from URL', error)
 
+          // hacky workaround to keep the existing session if there's an error returned from identity linking
+          // TODO: once error codes are ready, we should match against it instead of the message
+          if (
```

Comment:
Is this check working correctly? It's showing as `info` instead of `error` in the logs. Also should the message string be `400: Identity is already linked to another user` instead of `Identity is already linked to another user`?

![image](https://github.com/supabase/gotrue-js/assets/78710/0dfb8480-ea6f-4bf9-920f-0ab831435c4a)

I'm also getting a 401 on `/user`:
![image](https://github.com/supabase/gotrue-js/assets/78710/921cd243-04a0-4ef8-8eb6-a6ae913b3d41)

@J0

Reply:
Hey @chhuang,

Thanks for the detailed write up! Looking at this now